en/user_guide/content_display.rst

```diff
@@ -0,0 +1,76 @@
+Contents display
+================
+
+Once contributed, there is only one way to display a content: using content blocks in nodes. Open
+Orchestra comes with 3 block types able to render a content, but a developper can create his own block
+to match its need. The 3 standard Open Orchestra content blocks are:
+
+* Content list block
+* Content block
+* Configurable content block
+
+
+Content list block
+------------------
+This block renders in Front Office a list of contents matching criterias given by the setting of the
+block. A typical use may be to display a list of short previews of several articles, or a listing of
```

Comment:
A bit heavy to read, i would rephrase to something like "a typical use case may be the display of short previews for an article list"